scripts/utils/generate_strings.py

```diff
@@ -56,6 +56,11 @@ def parseTranslationStrings():
         exit("Unable to find translations/strings.yaml")
 
     with open(yaml_path, "r", encoding="utf-8") as yaml_file:
+        # Enforce a new line at the end of the file
```

Comment:
This is the actual change.

Reply:
Bah, I even thought of checking that, and then forgot 🤦🏼 

What about using `.seek()`. Trying with that, but happy to create two read cycles instead.

Reply:
`.seek()` works too.